content/en/docs/instrumentation/java/libraries.md

```diff
@@ -23,3 +23,180 @@ some instrumentation libraries for your dependencies standalone. To find out
 which standalone instrumentation libraries are available, take a look at
 [this list](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/docs/supported-libraries.md#libraries--frameworks).
 Follow the instructions of each instrumentation library to set them up.
+
+## Example app preparation {#example-app}
+
+We will instrument an HTTP client application using library instrumentation
+which will make a call to HTTP server.
+
+You can use the dice example app as HTTP server from
+[Getting Started](/docs/instrumentation/java/getting-started/) or you can create
+your own HTTP server.
+
+### Dependencies {#example-app-dependencies}
+
+To begin, set up an environment in a new directory called
+`java-simple-http-client`. Within that directory, create a file called
+`build.gradle.kts` with the following content:
+
+```kotlin
+plugins {
+  id("java")
+  id("application")
+}
+
+application {
+  mainClass.set("otel.SampleHttpClient")
+}
+
+sourceSets {
+  main {
+    java.setSrcDirs(setOf("."))
+  }
+}
+
+repositories {
+  mavenCentral()
+}
+
+dependencies {
+    implementation("io.opentelemetry:opentelemetry-api:1.32.0");
+    implementation("io.opentelemetry:opentelemetry-sdk:1.32.0");
+    implementation("io.opentelemetry:opentelemetry-exporter-logging:1.32.0");
+    implementation("io.opentelemetry.semconv:opentelemetry-semconv:1.22.0-alpha");
+    implementation("io.opentelemetry:opentelemetry-sdk-extension-autoconfigure:1.32.0");
+    implementation("io.opentelemetry.instrumentation:opentelemetry-java-http-client:1.31.0-alpha");
+}
+```
+
```

Comment:
Add a not here that `maven` can be used of course as well and that the relevant dependency here is `io.opentelemetry.instrumentation:opentelemetry-java-http-client:1.31.0-alpha`

Reply:
Added, Please check.

Reply:
@svrnm - shouldn't we use tabbed panes in this case instead?

Reply:
that would be nice here too: https://opentelemetry.io/docs/instrumentation/java/getting-started/#dependencies

Reply:
@chalin @trask I have improved with tabbed panes now, Please suggest if any more text need to add for same ?

Reply:
Tabbed pane seems file. I'll let @svrnm give the final work as to whether this comment is resolved to his liking.

> that would be nice here too: https://opentelemetry.io/docs/instrumentation/java/getting-started/#dependencies

@trask - maybe we can open a separate issue for this?

Reply:
@nerudadhich would it be a lot of effort to make this work with gradle and maven out of the box? If yes, let's keep it as is and have a separate issue for that, if no, feel free to update the PR accordingly. This would than nicely fit with the other existing documentation where we provide both

Reply:
Hi @svrnm I tired spending some time on this but I am not Maven expert if we can get someone's help would be great or I can still spend more time if we are not in rush.